pkg/projectfile/projectfile.go

```diff
@@ -67,7 +67,7 @@ var strReg = fmt.Sprintf(`https:\/\/%s\/([\w_.-]*)\/([\w_.-]*)(?:\?commitID=)*(.
 // ProjectURLRe Regex used to validate project fields /orgname/projectname[?commitID=someUUID]
 var ProjectURLRe = regexp.MustCompile(strReg)
 
-const localProjectsConfigKey = "projects"
+const LocalProjectsConfigKey = "projects"
```

Comment:
We gotta find a better solution than just exposing the key, but I also don't want to start making the projectfile or project package a catchall for ANYTHING project.. Let's just keep it in mind for now, don't want to add scope.